Caw.h

```diff
@@ -0,0 +1,20 @@
+#include <string>
+#include <gflags/gflags.h>
+
+DECLARE_string(user);
+DECLARE_string(registeruser);
+DECLARE_string(caw);
+DECLARE_string(reply);
+DECLARE_string(read);
+DECLARE_string(follow);
+DECLARE_bool(profile);
+
+class Caw
+{
+	public:
+	    static void registeruserHandler(std::string username); //
```

Comment:
Rename functions to start with a capital.

Also take const reference args.

Reply:
Renamed and arguments changed